packages/create-nuxt-app/lib/prompts.js

```diff
@@ -109,10 +109,19 @@ module.exports = [
     type: 'checkbox',
     choices: [
       { name: 'jsconfig.json (Recommended for VS Code if you\'re not using typescript)', value: 'jsconfig.json' },
-      { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' }
+      { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' },
+      { name: 'Dependabot (For auto-updating dependencies)', value: 'dependabot' }
```

Comment:
Dependabot only supports github and is going to github built-in soon. While still major share for private repo usage is for gitlab. (https://github.com/dependabot/dependabot-core/issues/399). In that case, i'm more in favor or having [renovate](https://github.com/renovatebot/renovate) option first but this PR should be valid as while as we mention it is for **github only**

Reply:
I most definitely can update the text to notify users that it'll work only on GitHub :)

Reply:
- [x] Fixed in 5cc555c